README.md

```diff
@@ -1,66 +1,81 @@
-Using the docker image (https://github.com/puckel/docker-airflow) please complete the following tasks.
+The goal of this assessment is for you to show us your data engineering experience, from designing to implementing a solution.
 
-## Task 1 - Getting Airflow up and running
-Since the tasks are simple, you can use LocalExecutor.
-docker-compose -f docker-compose-LocalExecutor.yml up -d
+This exercise consists of multiple phases. Ideally the end result covers all phases, however when you feel you're unable to complete a certain step you can pass and try to continue on the next.
 
-As a test, we expect you to get the docker container up and running.
-Once you are able to use it, as a small task, use airflow repository as a source and the same database as a target to create a table in the following structure:
- - table_name (string): table name in the airflow repository
- - row_number (int): number of rows within that table
+## Assessment
 
-There are 22 tables in the repository (dag_run, dag, connection, job, sla_miss, etc.).
+We'd like to analyze flight information for a set of transactions which are shared as data input files. Because we prefer sustainable applications and data pipelines we encourage you to think about scalability and maintainability. The input files are relatively small, but imagine we receive a tenfold of this data per day when designing a solution. 
 
-The deliverables of this task are 
- - a dag file for the pipeline
- - a csv of the data of the final table
+Your mission is to design and implement one or multiple applications or data pipelines to consume these files, store the data to an analytical store and deliver (some of) the requested insights. 
 
-| table_name   | row_count |
-| ------------ | --------- |
-| dag_run      | 0         |
-| dag_stats    | 0         |
-| import_error | 0         |
-| known_event  | 0         |
-| ...          |           |
+The obvious and most simple solution would be to load data into a spreadsheet or data platform of your choice manually, however we would be more impressed with a solution which includes technologies like Kubernetes, Docker, message queue or message broker (i.e. RMQ, Kafka; *we use Google Pub/Sub*), data processing (i.e. Spark; *we use Google DataFlow/Apache Beam*), orchestration (*we use Apache Airflow*), storage layer (i.e. HDFS, Redshift; *we use Google BigQuery*)
 
+We've described a possible approach in the phases below
 
-## Task 2 - Cleansing
-For this task, your input is the data_cleansing_input.json file. The data is in json format.
+### Phase 1
+Write a small applications which reads the data from all input files and published the data to a message broker like Kafka or Pub/Sub. Use a docker container to run the application.
```

Comment:
nitpicking: it should be "application", and "publishes"

Reply:
Thanks for reviewing @markvincze !